### PR TITLE
[julia/en] fix for #1483

### PR DESCRIPTION
**Pull request by kaymmm**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

---

Camel cased everything per the style spec to help clear out issue #1483

### CODE REVIEW

**Comment by divayprakash** [comment]
From the Julia [style guide](https://docs.julialang.org/en/v1.0/manual/style-guide/) -

> #### Use naming conventions consistent with Julia `base/`
> * modules and type names use capitalization and camel case: `module SparseArrays`, `struct UnitRange`.
> * functions are lowercase (`maximum`, `convert`) and, when readable, with multiple words squashed together (`isequal`, `haskey`). When necessary, use underscores as word separators. Underscores are also used to indicate a combination of concepts (`remotecall_fetch` as a more efficient implementation of `fetch(remotecall(...))`) or as modifiers.
> conciseness is valued, but avoid abbreviation (`indexin` rather than `indxin`) as it becomes difficult to remember whether and how particular words are abbreviated.
> 
> If a function name requires multiple words, consider whether it might represent more than one concept and might be better split into pieces.

File: julia.html.markdown [hunk 1]
```diff
@@ -434,8 +434,8 @@ add(5, 6)
 # => 11
 
 # Compact assignment of functions
-f_add(x, y) = x + y  # => f_add (generic function with 1 method)
-f_add(3, 4)  # => 7
+fAdd(x, y) = x + y  # => fAdd (generic function with 1 method)
+fAdd(3, 4)  # => 7
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 2]
```diff
@@ -478,67 +478,67 @@ catch e
 end
 
 # You can define functions that take keyword arguments
-function keyword_args(;k1=4, name2="hello")  # note the ;
+function keywordArgs(;k1=4, name2="hello")  # note the ;
     return Dict("k1" => k1, "name2" => name2)
 end
-# => keyword_args (generic function with 1 method)
+# => keywordArgs (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 3]
```diff
@@ -478,67 +478,67 @@ catch e
 end
 
 # You can define functions that take keyword arguments
-function keyword_args(;k1=4, name2="hello")  # note the ;
+function keywordArgs(;k1=4, name2="hello")  # note the ;
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 4]
```diff
@@ -478,67 +478,67 @@ catch e
...
 end
-# => keyword_args (generic function with 1 method)
+# => keywordArgs (generic function with 1 method)
 
-keyword_args(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
-keyword_args(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
-keyword_args()              # => ["name2"=>"hello", "k1"=>4]
+keywordArgs(name2="ness")  # => ["name2"=>"ness", "k1"=>4]
+keywordArgs(k1="mine")     # => ["name2"=>"hello", "k1"=>"mine"]
+keywordArgs()              # => ["name2"=>"hello", "k1"=>4]
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 5]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
 # You can combine all kinds of arguments in the same function
-function all_the_args(normal_arg, optional_positional_arg=2; keyword_arg="foo")
-    println("normal arg: $normal_arg")
-    println("optional arg: $optional_positional_arg")
-    println("keyword arg: $keyword_arg")
+function allTheArgs(normalArg, optionalPositionalArg=2; keywordArg="foo")
+    println("normal arg: $normalArg")
+    println("optional arg: $optionalPositionalArg")
+    println("keyword arg: $keywordArg")
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 6]
```diff
@@ -478,67 +478,67 @@ catch e
...
 end
-# => all_the_args (generic function with 2 methods)
+# => allTheArgs (generic function with 2 methods)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 7]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
-all_the_args(1, 3, keyword_arg=4)
+allAheArgs(1, 3, keywordArg=4)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 8]
```diff
@@ -478,67 +478,67 @@ catch e
...
 # => normal arg: 1
 # => optional arg: 3
 # => keyword arg: 4
 
 # Julia has first class functions
-function create_adder(x)
+function createAdder(x)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 9]
```diff
@@ -478,67 +478,67 @@ catch e
...
     adder = function (y)
         return x + y
     end
     return adder
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 10]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
 # This is "stabby lambda syntax" for creating anonymous functions
 (x -> x > 2)(3)  # => true
 
-# This function is identical to create_adder implementation above.
-function create_adder(x)
+# This function is identical to createAdder implementation above.
+function createAdder(x)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 11]
```diff
@@ -478,67 +478,67 @@ catch e
...
     y -> x + y
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 12]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
 # You can also name the internal function, if you want
-function create_adder(x)
+function createAdder(x)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 13]
```diff
@@ -478,67 +478,67 @@ catch e
...
     function adder(y)
         x + y
     end
     adder
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 14]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
-add_10 = create_adder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64}) 
+add10 = createAdder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64})
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 15]
```diff
@@ -478,67 +478,67 @@ catch e
...
     adder
 end
-# => create_adder (generic function with 1 method)
+# => createAdder (generic function with 1 method)
 
-add_10 = create_adder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64}) 
+add10 = createAdder(10) # => (::getfield(Main, Symbol("#adder#11")){Int64}) 
                           # (generic function with 1 method)
-add_10(3) # => 13
+add10(3) # => 13
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 16]
```diff
@@ -478,67 +478,67 @@ catch e
...
 
 
 # There are built-in higher order functions
-map(add_10, [1,2,3])  # => [11, 12, 13]
+map(add10, [1,2,3])  # => [11, 12, 13]
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 17]
```diff
@@ -478,67 +478,67 @@ catch e
...
 filter(x -> x > 5, [3, 4, 5, 6, 7])  # => [6, 7]
 
 # We can use list comprehensions
-[add_10(i) for i = [1, 2, 3]]   # => [11, 12, 13]
-[add_10(i) for i in [1, 2, 3]]  # => [11, 12, 13]
+[add10(i) for i = [1, 2, 3]]   # => [11, 12, 13]
+[add10(i) for i in [1, 2, 3]]  # => [11, 12, 13]
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 18]
```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
 Panther <: Cat  # => true
 
 # Defining a function that takes Cats
-function pet_cat(cat::Cat)
+function petCat(cat::Cat)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 19]
```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
...
     println("The cat says $(meow(cat))")
 end
-# => pet_cat (generic function with 1 method)
+# => petCat (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 20]
```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
...
 
-pet_cat(Lion("42")) # => The cat says 42
+petCat(Lion("42")) # => The cat says 42
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 21]
```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
...
 try
-    pet_cat(tigger) # => ERROR: MethodError: no method matching pet_cat(::Tiger)
+    petCat(tigger) # => ERROR: MethodError: no method matching petCat(::Tiger)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 22]
```diff
@@ -744,14 +744,14 @@ fight(Lion("RAR"), Lion("brown", "rarrr"))  # => The lions come to a tie
 # Under the hood
 # You can take a look at the llvm  and the assembly code generated.
 
-square_area(l) = l * l  # square_area (generic function with 1 method)
+squareArea(l) = l * l  # squareArea (generic function with 1 method)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 23]
```diff
@@ -744,14 +744,14 @@ fight(Lion("RAR"), Lion("brown", "rarrr"))  # => The lions come to a tie
...
 
-square_area(5)  # => 25
+squareArea(5)  # => 25
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 24]
```diff
@@ -744,14 +744,14 @@ fight(Lion("RAR"), Lion("brown", "rarrr"))  # => The lions come to a tie
...
 
-# What happens when we feed square_area an integer?
-code_native(square_area, (Int32,), syntax = :intel)
+# What happens when we feed squareArea an integer?
+codeNative(squareArea, (Int32,), syntax = :intel)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 25]
```diff
@@ -744,14 +744,14 @@ fight(Lion("RAR"), Lion("brown", "rarrr"))  # => The lions come to a tie
...
 	#         .text
-	# ; Function square_area {
+	# ; Function squareArea {
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 26]
```diff
@@ -832,9 +832,9 @@ code_native(circle_area, (Int32,), syntax = :intel)
     #         nop     dword ptr [rax]
     # ;}
 
-code_native(circle_area, (Float64,), syntax = :intel)
+codeNative(circleArea, (Float64,), syntax = :intel)
     #         .text
-    # ; Function circle_area {
+    # ; Function circleArea {
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 27]
```diff
@@ -832,9 +832,9 @@ code_native(circle_area, (Int32,), syntax = :intel)
     #         nop     dword ptr [rax]
     # ;}
 
-code_native(circle_area, (Float64,), syntax = :intel)
+codeNative(circleArea, (Float64,), syntax = :intel)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 28]
```diff
@@ -798,12 +798,12 @@ code_native(square_area, (Float64,), syntax = :intel)
...
-circle_area(r) = pi * r * r     # circle_area (generic function with 1 method)
-circle_area(5)  # 78.53981633974483
+circleArea(r) = pi * r * r     # circleArea (generic function with 1 method)
+circleArea(5)  # 78.53981633974483
 
-code_native(circle_area, (Int32,), syntax = :intel)
+codeNative(circleArea, (Int32,), syntax = :intel)
     #         .text
-    # ; Function circle_area {
+    # ; Function circleArea {
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 29]
```diff
@@ -798,12 +798,12 @@ code_native(square_area, (Float64,), syntax = :intel)
 # Note that julia will use floating point instructions if any of the
 # arguments are floats.
 # Let's calculate the area of a circle
-circle_area(r) = pi * r * r     # circle_area (generic function with 1 method)
-circle_area(5)  # 78.53981633974483
+circleArea(r) = pi * r * r     # circleArea (generic function with 1 method)
+circleArea(5)  # 78.53981633974483
 
-code_native(circle_area, (Int32,), syntax = :intel)
+codeNative(circleArea, (Int32,), syntax = :intel)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 30]
```diff
@@ -798,12 +798,12 @@ code_native(square_area, (Float64,), syntax = :intel)
 # Note that julia will use floating point instructions if any of the
 # arguments are floats.
 # Let's calculate the area of a circle
-circle_area(r) = pi * r * r     # circle_area (generic function with 1 method)
-circle_area(5)  # 78.53981633974483
+circleArea(r) = pi * r * r     # circleArea (generic function with 1 method)
+circleArea(5)  # 78.53981633974483
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 31]
```diff
@@ -780,9 +780,9 @@ code_native(square_area, (Float32,), syntax = :intel)
     #         nop     word ptr [rax + rax]
     # ;}
 
-code_native(square_area, (Float64,), syntax = :intel)
+codeNative(squareArea, (Float64,), syntax = :intel)
     #         .text
-    # ; Function square_area {
+    # ; Function squareArea {
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 32]
```diff
@@ -780,9 +780,9 @@ code_native(square_area, (Float32,), syntax = :intel)
     #         nop     word ptr [rax + rax]
     # ;}
 
-code_native(square_area, (Float64,), syntax = :intel)
+codeNative(squareArea, (Float64,), syntax = :intel)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 33]
```diff
@@ -765,9 +765,9 @@ code_native(square_area, (Int32,), syntax = :intel)
 	#         nop     dword ptr [rax + rax]
 	# ;}
 
-code_native(square_area, (Float32,), syntax = :intel)
+codeNative(squareArea, (Float32,), syntax = :intel)
     #         .text
-    # ; Function square_area {
+    # ; Function squareArea {
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

File: julia.html.markdown [hunk 34]
```diff
@@ -765,9 +765,9 @@ code_native(square_area, (Int32,), syntax = :intel)
 	#         nop     dword ptr [rax + rax]
 	# ;}
 
-code_native(square_area, (Float32,), syntax = :intel)
+codeNative(squareArea, (Float32,), syntax = :intel)
```
Comment (divayprakash) [inline julia.html.markdown]: Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194

**Comment by divayprakash** [comment]
@kaymmm Let me know once you're done and I'll merge this in. Great effort from you!

**Comment by divayprakash** [comment]
A function named `codeNative` occurring a few times after line 750 is all that remains now I think.

**Comment by kaymmm** [comment]
Great catch--I need to install a Julia style linter!

**Comment by divayprakash** [comment]
@kaymmm Any progress on this?

**Comment by kaymmm** [comment]
I believe that I already fixed all of the requested changes in the last commit, including the codeNative fixes. Let me know if I missed anything.

**Comment by divayprakash** [comment]
@kaymmm Thank you for putting in the work on this!

**Pull request merged**